src/include/job.h

```diff
@@ -967,7 +967,6 @@ task_find	(job		*pjob,
 #define JOB_STDERR_SUFFIX  ".ER"	/* job standard error */
 #define JOB_CKPT_SUFFIX    ".CK"	/* job checkpoint file */
 #define JOB_TASKDIR_SUFFIX ".TK"	/* job task directory */
-#define JOB_CPUSET_SUFFIX  ".CS"	/* job cpuset */
 #define JOB_BAD_SUFFIX	   ".BD"	/* save bad job file */
```

Comment:
Indentation is off here.

Reply:
It seems to be a Github rendering thing.  It lines up in the actual file.

Reply:
The problem is there is a tab on that line. The other lines use spaces.